librpc/Rpc.cpp

```diff
@@ -57,7 +57,8 @@ std::map<int, std::string> dev::rpc::RPCMsg{{RPCExceptionType::Success, "Success
     {RPCExceptionType::InvalidRequest,
         "Don't send request to this node who doesn't belong to the group"},
     {RPCExceptionType::IncompleteInitialization, "RPC module initialization is incomplete."},
-    {RPCExceptionType::OverQPSLimit, "Over QPS limit"}};
+    {RPCExceptionType::OverQPSLimit, "Over QPS limit"},
+    {RPCExceptionType::PermissionDenied, " PermissionDenied for not in the allowlist"}};
```

Comment:
The SDK is not allowed to access this group. maybe more clear

Reply:
modified.